qiskit/optimization/converters/inequality_to_equality.py

```diff
@@ -234,12 +232,11 @@ def _add_auto_slack_var_linear_constraint(self, linear, sense, rhs, name):
             self._add_integer_slack_var_linear_constraint(linear, sense, rhs, name)
 
     def _add_integer_slack_var_quadratic_constraint(self, linear, quadratic, sense, rhs, name):
-        # If a coefficient that is not integer exist, raise an error
+        # If a coefficient that is not integer exist, raise error
         if any(
```

Comment:
Similar float check appears often in the class. So, I suggest to introduce a static method.
```python
@staticmethod
    def _contains_any_float_value(values: List[Union[int, float]]) -> bool:
        return any(isinstance(val, float) and not val.is_integer() for val in values)
```